index.js

```diff
@@ -31,17 +31,34 @@ module.exports = {
 	},
 
 	/**
+	 * Remove any handlers
+	 * @returns {exports}
+	 */
+	reset: function() {
+		MockXMLHttpRequest.handlers = [];
```

Comment:
Can this just proxy to `lib/MockXMLHttpRequest.js`'s `reset()` rather than re-implementing the logic